src/checkers/athena.rs

```diff
@@ -0,0 +1,49 @@
+use crate::checkers::checker_result::CheckResult;
+use lemmeknow::Identify;
+
+use super::{
+    checker_type::{Check, Checker},
+    english::EnglishChecker,
+    human_checker,
+    lemmeknow_checker::LemmeKnow,
+};
+
+/// Athena checker runs all other checkers
```

Comment:
Any chance we can get doctests like we do with decoders? https://github.com/bee-san/Ares/blob/main/src/decoders/base64_decoder.rs#L18-L24

Reply:
Hopefully it doesn't compile fail like the one I linked haha

Maybe I should do this in another PR!

Reply:
I have updated that base64 test, new one doesn't fail to compile <3 ( and it works too haha )